encryption.rst

```diff
@@ -171,6 +171,10 @@ To ``run``, ``shell``, or ``exec`` an encrypted image, credentials to
 decrypt the image need to be supplied at runtime either in a key-file or
 a plaintext passphrase.
 
+.. note:: 
+
+   Only when running with setuid installation of {Project}, ``--userns`` option is required.
```

Comment:
But that's not required when a container is encrypted by a root user, right?

We need more elaboration on the fact that there's two different file formats for encryption, one which is used when a container is built by the root user and another when built by an unprivileged user.  Tell people how to tell the difference by using `apptainer sif show`.  Tell people that to decrypt the type made by root, you need to either be root or be using a setuid-root installation of {Project}, and that if you want to decrypt the type not made by root, with a setuid-root installation you need the `--userns` option.

Reply:
hmm, I add some explanation to different types of images. 
https://github.com/apptainer/apptainer-userdocs/pull/179/commits/0c5d93d3ca2a71b39fa0db2bcb5e66ef7dbc3671

Reply:
I don't see it.  Maybe it didn't get into the commit somehow?  The rest of the comments are still unaddressed.